packages/types/src/codec/Struct.ts

```diff
@@ -111,6 +111,13 @@ export default class Struct<
     return class extends Struct<S> {
       constructor (value?: any, jsonMap?: Map<keyof S, string>) {
         super(Types, value, jsonMap);
+
+        (Object.keys(Types) as Array<keyof S>).forEach((key, index) => {
+          Object.defineProperty(this, key, {
+            enumerable: true,
+            get: () => this.getAtIndex(index)
```

Comment:
`this.get(key)` is faster, we don't do an .toArray() each time.

Reply:
Yes, actually had it as `get(key)` originally, will revert. Thanks.